aria-practices.html

```diff
@@ -3734,6 +3734,390 @@ <h5>Examples</h5>
+          <li><code>all</code>: Synonym to <code>additions removals text</code></li>
+        </ul>
+
+        <p>
+          If <code>aria-relevant</code> is not specified, then the value of the closest ancestor element with an <code>aria-relevant</code> attribute is used.
+          Specifying the <code>aria-relevant</code> attribute on an element overrides any value specified on an ancestor element.
+          If there is no ancestor element with an <code>aria-relevant</code> attribute, the default value <code>additions text</code> is used.
+        </p>
+
+        <p>
+          For example, a disappearing old message in a chat log is not significant, and users do not need to be informed of the removal.
+          However, for a list of online contacts, a disappearing contact is significant (it indicates that the contact is no longer online).
+          Instead of announcing the removal of something, it is often better to add new content that tells the user what happened.
+          For example, "Alice is now offline" is clearer than "Alice" or "Removed, Alice".
+          Therefore, avoid using <code>removals</code> or <code>all</code>.
+        </p>
+
+        <p>
+          For example, a list of online contacts:
+        </p>
+
+        <pre><code>&lt;div role="region" aria-live="polite" aria-labelledby="contacts">
+ &lt;h1 id="contacts">Contacts&lt;/h1>
+ &lt;ul>
+  &lt;li aria-atomic="true">&lt;a href="/contacts/alice">Alice&lt;/a> is now online&lt;/li>
+ &lt;/ul>
+&lt;/div>
+</code></pre>
+
+        <p>
+          Note that this example omits the <code>aria-relevant</code> attribute, leaving it as the default value (<code>additions text</code>).
+        </p>
+
+        <p>
+          When a contact comes online, it is added to the list, and users of assistive technology are informed of the addition without disrupting their current task.
+          To communicate what happened, the text is "Alice is now online", rather than only "Alice".
+        </p>
+
+        <p>
+          Similarly when a user goes offline, the text could be changed to "Alice is now offline",
+          and then be hidden from the list after a timeout.
+        </p>
+
+        <p>
+          If a contact changes their display name, the text change would also be announced:
+          "Alice has changed their name to 4liz".
+          After a timeout, the text "Alice has changed their name to " could be removed, without causing a new announcement.
+        </p>
+      </section>
+
+      <section id="triggering_live_regions">
+        <h4>Triggering Live Regions</h4>
+
+        <p>
+          Additions and removals in the accessibility tree can happen due to changes to the DOM tree or changes to the applied CSS.
+          For example, changing the CSS <code>display</code> property to <code>none</code> causes the element to be removed from the accessibility tree.
+          See the <a href="#accessibility-tree">Accessibility tree</a> section for more details.
```

Comment:
This section needs a bit more discussion. Does setting `display: block;` on the live region trigger it? Or does it have to be rendered first, and then have a change? Does changing CSS display on live region descendants trigger it?